crates/op-rbuilder/src/payload_builder_vanilla.rs

```diff
@@ -1355,3 +1410,88 @@ fn estimate_gas_for_builder_tx(input: Vec<u8>) -> u64 {
 
     zero_cost + nonzero_cost + 21_000
 }
+
+/// Extracts commitment from access list entries, pointing to 0x420..022 and validates them
+/// against supervisor.
+///
+/// If commitment present pre-interop tx rejected.
+///
+/// Returns (is_valid, is_recoverable)
```

Comment:
What exactly is `is_recoverable`? Is it like an RPC error?

Reply:
we remove totally invalid transactions from mempool
And if the transaction is not yet valid - just skip it for the current round